apps/simple/pinecone-vector-store/README.md

```diff
@@ -0,0 +1,33 @@
+# Pinecone Vector Store
```

Comment:
please move example to `/examples/` and try with the local `llamaindex` build (see https://github.com/run-llama/LlamaIndexTS/blob/main/examples/README.md#using-the-local-build)